src/main/java/org/mcmonkey/denizen2core/commands/queuecommands/AssertCommand.java

```diff
@@ -20,7 +20,7 @@
     // @Minimum 2
     // @Maximum 2
     // @Description
-    // Throws an error if the specified boolean is false.
+    // Throws an error if the specified boolean is false, stopping the queue.
```

Comment:
Unnecessary here, this fact should be documented elsewhere and linked to.

A general explanation of what a script error is and how they function in D2.